src/components/__tests__/HelloWorld.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
I just followed former structure of this project, but I'm for co-locating of tests with components

Reply:
Right, this is the trend I've been observing in many React projects.

Co-locating `.test.js` and `.stories.js` and other React component "friends" makes it easier to navigate the project structure, improving developer productivity. We should also make it a rule that every React component has an associated unit test, this can be done through validations.

Reply:
I'm glad we're doing this. I've seen it in other projects and really like it as opposed to maintaining a completely separate directory for tests.